1pr_build_requires.py

```diff
@@ -4,7 +4,8 @@
 from conans.test.utils.tools import TestClient
 from conans import tools
 
-me = os.path.abspath(os.path.dirname(__file__))
+me = os.path.abspath(os.path.dirname(__file__)).replace("\\", "/")
```

Comment:
why? is it needed?

Reply:
Yes, otherwise in Windows fails.

Reply:
why? how?

`conan export C:\\...\\path/recipe/conanfile.py` fails?

Reply:
No, not sure where it is, somewhere inside ``TestClient`` I guess, didn't investigate further.

Reply:
You have also removed the ``working_dir`` management? It is also a bit annoying that it fails and you need to create it manually.

Reply:
It should work, the TestClient creates that folder if it doesn't exist